components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -112,6 +113,8 @@ class RewardsServiceImpl : public RewardsService,
       std::unique_ptr<RewardsServicePrivateObserver> private_observer,
       std::unique_ptr<RewardsNotificationServiceObserver>
           notification_observer);
+  void PreferenceObserver();
+  void OnPreferenceChanged(const std::string& key);
```

Comment:
Seems like these could be private methods.

Reply:
fixed